doc/md/fundamentals/3-declarations/1-variable-declarations.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 1
+---
+
+# Variable declarations
+
+In Motoko, variables are declared using either `let` (immutable) or `var` (mutable).
+
+| Feature                 | `let`    | `var`       |
+|-------------------------|---------|-------------|
+| Reassignment           | Not allowed | Allowed |
+| Value modification     | Cannot be updated after assignment. | Can be updated after assignment. |
+| Memory allocation      | Directly stores value. | Stores a reference to a memory location. |
+| Persistence in actors  | Retained unless assigned a mutable value. | Allowed with `stable var`. |
+| Upgrade behavior       | Persists unless holding a `var` reference. | Must be explicitly marked `stable` to persist. |
+| Compiler optimization  | More optimal | Less optimal |
+
+## Immutable variables
+
+A `let` declaration binds a value to a name and cannot be changed after assignment. The compiler ensures that any attempt to modify an immutable variable results in an error.
```

Comment:
```suggestion
A `let` declaration names the value of an expression so that the expression can be evaluated once and its result, a value, referenced by that name. The compiler ensures that any attempt to modify an immutable variable results in an error.
```